bundles/org.openhab.binding.carnet/README.md

```diff
@@ -0,0 +1,248 @@
+# CarNet Binding
+
+This binding integrates the CarNet service provided by Audi, Volkswagen and other brands.
+It provides access to status information and control functions for compatible vehicles, which have activated the service.
+The API provides a lot of diagnostic data and provide access to remote control functions like lock/unlock door.
+
+## Discovery
+
+The binding supports automated discovery of vehicles.
+For now only Audi is supported, other brand might follo w in the future, because the API has a big overlap between various brands.
+The CarNet Account Thing has to be added manually giving the neccesary credentials.
+Once the account is online the binding can query all registered vehicles and creates a vehicle thing for each of them.
+
+## Binding Configuration
+
+The binding itself has no configuration options
+
+## Supported Things
+
+### CarNet Account (account)
+
+The Account thing implements the online connect to the CarNet service.
+An account is required to setup the connection (register on the manufacture's portal, e.g. myAudi.de).
+Usually the manufacturer supports to register a single or multiple vehicles under this account (identified by their unique identification number - VIN).
+Once the account gets online the binding retrieves the complete vehicle list and creates a thing per vehicle.
+
+Thing Configuration
+
+|Parameter         | Description                                                                               |Mandatory| Default   |
+|------------------|-------------------------------------------------------------------------------------------|---------|-------------------|
+| brand            | Car brand, currently only Audi is supported                                               | yes     | none              |
```

Comment:
No really, the brand becomes are of the API URL (beside market) and will be used to handle some brand specific stuff like OAuth flow. Following "A vehicle is a vehicle just from different brands" I see no value in having different things types - would be just a different technical representation.  

Current the user flow is
- Add a CarNet Account: set brand and credentials
- run auto-discovery: finds all vehicles under this brand
maybe your approach is more expressive for the user
- Add "myAudi Account" vs. "myVolgswagen Account"

I think I'll go that way

Reply:
From the usability perspective I think it's easier to choose the correct Bridge for you model instead of configuring it afterwards.

What matters more from my perspective is to maintain the object orientented program paradigm of using polymorphism. If there are different functionalities to be handled you could map them easily between Things and (inherited) Handlers. Implementing different behaviors by branching (`if`) is against the OOP approach.

Reply:
I changed to account thing "myaudi", "myvolkswagen" will be follow when this brand is added